FunctionalTableData/TableSection.swift

```diff
@@ -20,7 +20,7 @@ public protocol TableSectionType {
 	var rows: [CellConfigType] { get }
 	/// Action to perform when the header view comes in or out of view.
 	var headerVisibilityAction: ((_ view: UIView, _ visible: Bool) -> Void)? { get }
-	@available(*, deprecated: 1.0, message: "Use `rows.count` instead.")
+	@available(iOS, deprecated: 1.0, message: "Use `rows.count` instead.")
```

Comment:
I don't know why, but Swift now hates "*" and a deprecated version.
Does this actually mean it's been deprecated since iOS 1.0?

Reply:
does this work on older Xcode versions? Wondering if we are forcing 10.2 on everyone now.

Reply:
Yeah, it'll come up as "`function` has been deprecated since iOS 1.0", I've seen this before.

Reply:
Looks like you could just omit the version number altogether: `@available(iOS, deprecated, message: "message")`

Other options here: https://docs.swift.org/swift-book/ReferenceManual/Attributes.html

Reply:
Builds on 10.1 👍

Reply:
I had that doc open @josephroquedev. It seems to imply that it was fine the way it's been for the last [value] years. 
I'll remove the version number and put back the *.

* == iOS according the build targets  😉